build-packages.sh

```diff
@@ -9,12 +9,15 @@ usage()
 
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 build_packages_log=$working_tree_root/build-packages.log
-arguments="$@"
+binclashlog=$working_tree_root/binclash.log
+binclashloggerdll=$working_tree_root/Tools/Microsoft.DotNet.Build.Tasks.dll
 
-echo -e "Running build-packages.sh $arguments" > $build_packages_log
+options="/nologo /v:minimal /clp:Summary /flp:v=detailed;Append;LogFile=$build_packages_log /l:BinClashLogger,$binclashloggerdll;LogFile=$binclashlog"
```

Comment:
Actually /m started working on Monday, so I'll update build-packages.sh to include it.
Thanks Nate!